src/resources/i18n/microsoft/Messages_fr_FR.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <translations>
 
-   <translation id="microsoft_office_license_officeProPlusofficeBusiness" qtlid="253218">Office Business</translation>
-   <translation id="microsoft_office_license_officeProPlusofficeProPlus" qtlid="253231">Office Pro Plus</translation>
+   <translation id="microsoft_office_license_office-business-pp" qtlid="253218">Office Business</translation>
+   <translation id="microsoft_office_license_office-proplus-pp" qtlid="253231">Office Pro Plus</translation>
+   <translation id="microsoft_office_license_officeBusiness" qtlid="253218">Office Business</translation>
```

Comment:
Duplicates.

Reply:
I know... but if you find a better way...
One is fetched from licenseEnum in the json schema, the other is fetched from the API response.